README.md

```diff
@@ -60,16 +60,20 @@ To test against a local hydra instance
     ```bash
     make start
     ```
+- You can also set your preset frontend URLs with environment variables, making them available in the admin UI. This allows you to switch between them seamlessly:
+    ```bash
+    RAZZLE_DEFAULT_IFRAME_URL=http://localhost:3002,https://hydra-blogsite-nextjs.vercel.app pnpm start
```

Comment:
whats the "pnpm start" for? mistake?

Reply:
Its intended, if we want to pass env vars, it was discussed with Jeff